app/models/user.rb

```diff
@@ -1,17 +1,34 @@
 class User < ApplicationRecord
+  attr_accessor :remember_token
   before_save :email_downcase
   has_secure_password
-
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
-
   validates :email, presence: true, length: {maximum: 255},
     format: {with: VALID_EMAIL_REGEX},
     uniqueness: {case_sensitive: false}
   validates :name, presence: true, length: {maximum: 50}
   validates :password, presence: true, length: {minimum: 6}
-
-  private
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.degest(remember_token)
+  end
   def email_downcase
```

Comment:
Để private nó báo lỗi : private method `remember' called for #<User:0x007fe671f4e750> lúc login